qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/car/MainCarSession.kt

```diff
@@ -60,7 +60,13 @@ class MainCarSession : Session() {
                             .build()
                     )
                 }
-                mapboxCarMap.setup(carContext, MapInitOptions(context = carContext))
+                mapboxCarMap.setup(
+                    carContext,
+                    MapInitOptions(
+                        context = carContext,
+                        styleUri = carMapLoader.getCurrentExtension(carContext.isDarkMode).styleUri
```

Comment:
Maybe we should set `null` initially, because with your implementation we don't set the actual style, only its uri. On the other hand, it probably doesn't matter what we set here, because according to the docs this style will be applied only in `onStart` and by that time `MapboxCarMapLoader` will have already loaded its own style.

Reply:
Yeah, I'll look into this again. Maybe it is not needed

Reply:
https://github.com/mapbox/mapbox-maps-android/blob/395d26231249a69973cbc3f9ea0553a6dff4119f/sdk/src/main/java/com/mapbox/maps/MapController.kt#L143-L149
```
    if (!mapboxMap.isStyleLoadInitiated) {
      // Load the style in mapInitOptions if no style has loaded yet.
      mapInitOptions.styleUri?.let {
        mapboxMap.loadStyleUri(it)
      }
    }
    pluginRegistry.onStart()
```

Yeah it's essentially this in the `MapController`. The mapInitOptions will be removed when the style load happens before `onStart`. In this case the style initialization in started in `onCreate`

👍